enterprise/internal/insights/background/queryrunner/work_handler_test.go

```diff
@@ -453,7 +455,7 @@ func TestGetSeries(t *testing.T) {
 	}
 
 	t.Run("series definition does not exist", func(t *testing.T) {
-		_, err := workHandler.getSeries(ctx, "seriesshouldnotexist")
+		_, err := workHandler.getSeries(ctx, log, "seriesshouldnotexist")
```

Comment:
We should provide loggers per subtest @davejrt : 

```suggestion
		_, err := workHandler.getSeries(ctx, logtest.Scoped(t), "seriesshouldnotexist")
```